modules/instance-template/variables.tf

```diff
@@ -49,6 +49,12 @@ variable "container" {
   type        = string
 }
 
+variable "env" {
+  description = "Environment variables to set to a VM that runs container."
+  type        = map(string)
```

Comment:
Please doublecheck the type the `env` variable, accepted by the [container-vm](https://registry.terraform.io/modules/terraform-google-modules/container-vm/google/latest#usage) Terraform module. According to the example they expect a list of objects of the following form:
```
variable "env" {
  description = "Environment variables to set to a VM that runs container."
  type        = list(object({ name = string, value = string }))
  default     = []
}
```